website/blog/2017-09-25-testing-rss.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Adding RSS Support - RSS Truncation Test
```

Comment:
Same issue here, why are we adding all of this boilerplate?

Reply:
refer to https://github.com/Microsoft/fast-dna/pull/1175#discussion_r237646930